README.md

```diff
@@ -1,2 +1,5 @@
-# Personal_Code_Review
-I create this repository to review my personal code while going through pair programming to join Microverse.
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please your ReadMe is not descriptive and you did not use Microverse provided template for your readme. You can try to reach out for checking the new Microverse readme template to make your project more descriptive.

Reply:
Please ensure to always state explicit reasons why you requested for a change; it might be obvious to you and not so for others. 😃 Moreso, it would be great if you can add the link to the new Microverse README template. 👍

Reply:
- [Optional] Great Job so far, I think it is better to use a couple of emojis and/or gifs in the review to make the code review more human-friendly as your precious code reviewer suggested you👽🤖